README.md

```diff
@@ -115,74 +141,60 @@ See [the example section](#examples-of-use) for usage examples.
 
           $ node build.js modules=interaction
 
-    2.2 You can also include all modules like so:
+    1.2 You can also include all modules like so:
 
           $ node build.js modules=ALL
 
-    2.3 You can exclude a few modules like so:
+    1.3 You can exclude a few modules like so:
 
           $ node build.js modules=ALL exclude=gestures,image_filters
 
-3. Create a minified distribution file
+2. Create a minified distribution file
 
         # Using YUICompressor (default option)
         $ node build.js modules=... minifier=yui
 
         # or Google Closure Compiler
         $ node build.js modules=... minifier=closure
 
-4. Enable AMD support via require.js (requires uglify)
+3. Enable AMD support via require.js (requires uglify)
 
         $ node build.js requirejs modules=...
 
-5. Create source map file for better productive debugging (requires uglify or google closure compiler).<br>More information about [source maps](http://www.html5rocks.com/en/tutorials/developertools/sourcemaps/).
+4. Create source map file for better productive debugging (requires uglify or google closure compiler).<br>More information about [source maps](http://www.html5rocks.com/en/tutorials/developertools/sourcemaps/).
 
         $ node build.js sourcemap modules=...
 
     If you use google closure compiler you have to add `sourceMappingURL` manually at the end of the minified file all.min.js (see issue https://code.google.com/p/closure-compiler/issues/detail?id=941).
 
         //# sourceMappingURL=fabric.min.js.map
 
-6. Ensure code guidelines are met (prerequisite: `npm -g install eslint`)
+5. Ensure code guidelines are met (prerequisite: `npm -g install eslint`)
 
         $ npm run lint && npm run lint_tests
 
 <h3 id="fabric-building">Testing</h3>
 
-1. [Install Node.js](https://github.com/joyent/node/wiki/Installation)
-
-2. [Install NPM, if necessary](https://github.com/npm/npm#super-easy-install)
-
-3. Install NPM packages
-
-        $ npm install
+#### 1. Install NPM packages
```

Comment:
Well the testing instructions are bad.
I think it's better to refer to CONTRIBUTING.md